spec/services/trivia_service_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe TriviaService do
+  describe 'conn' do
+    it 'has a connection to opentdb' do
+      conn = TriviaService.conn
+      trivia = TriviaFacade.trivia
```

Comment:
don't need this line here if you're not doing anything with it. that method should probably be tested but maybe inside of a new `it` block